### PR TITLE
Eff

### PR DESCRIPTION
**Pull request by liz-fw**

### CODE REVIEW

**Review by ErinCain — APPROVED** [review]

File: vignettes/EFF.Rmd [hunk 1]
```diff
@@ -6,22 +6,25 @@ output: html_document
...
+
+We call this modified EFF approach a Constructed Salmonid Flow (CSF).
 
-**Pilot EFF Approach on the Sacramento River**
+**Pilot CSF Approach on the Sacramento River**
 
-The R2R technical team is working collaboratively with a flow sub group to develop a EFF hydrograph for the Sacramento River. We are building off of the Nature Conservancy's Natural Flows Database to develop a wet and dry year monthly EFF hydrographs. We will compile these wet and dry year EFF into a 20 year synthetic timeserries based on the historical water year types between 1980 and 2000, the current R2R model timeframe.
+The R2R technical team is worked collaboratively with a flow subgroup to develop a CSF hydrograph for the Sacramento River. We built off of the Nature Conservancy's Natural Flows Database to develop a wet and dry year monthly EFF hydrograph and compiled these into a 20 year synthetic time series based on the historical water year types represented in the R2R model time frame (1980-2000).
```
[the remark this answers is not in the document]
Comment (liz-fw) [reply vignettes/EFF.Rmd:19]: addressed

File: vignettes/EFF.Rmd [hunk 2]
```diff
@@ -37,25 +40,26 @@ For more information on EFF please refer to additional sources:
...
+The R2R techncial team developed proposed CSF monthly flows by combining the [Natural Flows Database](https://ceff.ucdavis.edu/sites/g/files/dgvnsk5566/files/media/documents/CEFF%20Technical%20Report%20Ver%201.0%20Mar_31_2021_DRAFT_FINAL%20for%20web.pdf) values with critical flow to survival migratory thresholds on the Sacramento River (from [Michel 2021](https://esajournals.onlinelibrary.wiley.com/doi/full/10.1002/ecs2.3498)). 
 
-1) Optimize spawning habitat during full spawning timeframe (October, November, and December)
+This original attempt did not maximize habitat so the technical team revised their approach to maximize modeled habitat and survival using the following principles: 
+
+1) Optimize spawning habitat during full spawning time frame (October, November, and December)
```
[the remark this answers is not in the document]
Comment (liz-fw) [reply vignettes/EFF.Rmd:51]: updated to ref here: https://reorienting-to-recovery.github.io/DSMhabitat/reference/sacramento_habitat.html

File: vignettes/EFF.Rmd [hunk 3]
```diff
@@ -6,22 +6,25 @@ output: html_document
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
 library(tidyverse)
-
 ```
 
 ## Incorperating Effective Functional Flows into R2R
```
[the remark this answers is not in the document]
Comment (liz-fw) [reply vignettes/EFF.Rmd:11]: done

File: vignettes/EFF.Rmd [hunk 4]
```diff
@@ -37,25 +40,26 @@ For more information on EFF please refer to additional sources:
 
 -   [Natural Flows Database](https://ceff.ucdavis.edu/sites/g/files/dgvnsk5566/files/media/documents/CEFF%20Technical%20Report%20Ver%201.0%20Mar_31_2021_DRAFT_FINAL%20for%20web.pdf)
 
-### How can we tailor EFF to our system
+### Tailoring EFF to our system
 
-EFF are hypothesized to be the best flows for salmon in our historical system. However, our system is highly managed so we may need to alter EFF slightly from their historical values to maximize benefit to salmon.
+EFF are hypothesized to be the best flows for salmon in our historical system. However, our system is highly managed so altered EFF slightly from their historical values to maximize benefit to salmon (creating what we are calling Constructed Salmonid Flows).
```
[the remark this answers is not in the document]
Comment (liz-fw) [reply vignettes/EFF.Rmd:45]: done

**Comment by liz-fw** [comment]
@ergz if you get a chance could you review this? I mostly just want eyes on the `transform_inputs_complete.R`

**Pull request closed without merging**